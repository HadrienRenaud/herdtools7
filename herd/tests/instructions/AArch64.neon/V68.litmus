AArch64 V68
{
uint64_t t[2] = {0xffffffffffffffff,0x0};
uint64_t u[2] = {0x1,0x0};
0:X0=t; 0:X1=u;
}
  P0                   ;
 LD1 {V0.2D},[X0]      ;
 LD1 {V1.2D},[X1]      ;
 ADD V0.2D,V0.2D,V1.2D ;
 ST1 {V0.2D},[X0]      ;

forall t={0,0}
