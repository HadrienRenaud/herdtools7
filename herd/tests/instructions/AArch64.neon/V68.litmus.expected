Test V68 Required
States 1
t={0,0};
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (t={0,0})
Observation V68 Always 1 0
Hash=770eeca30938fabdcbbe69ff224d56f3

